SessionMessagingKit/Sending & Receiving/Attachments/TSAttachment.h

```diff
@@ -23,7 +23,7 @@ typedef NS_ENUM(NSUInteger, TSAttachmentType) {
 // The attachmentSchemaVersion and serverId properties only apply to
 // TSAttachmentPointer, which can be distinguished by the isDownloaded
 // property.
-@property (atomic, readwrite) UInt64 serverId;
+@property (atomic, readwrite) NSString *serverId;
```

Comment:
@RyanRory Just an FYI - this change will result in old versions of the app crashing if the user sent an attachment in a new version and downgraded to and old version (There is a comment explaining this change in `FileUploadResponse` but basically the server guys would like to be able to eventually migrate to string ids instead of int ones)

Reply:
Actually, I'll change this back - looks like this is based on a Protobuf config anyway so